content/05-expression.md

```diff
@@ -323,6 +323,7 @@ Operator | Operation | Operand 1 | Operand 2 | Result type
 <code>*=</code> | multiplication | `Float/Int` | `Float/Int` | `Float/Int` 
 <code>/=</code> | division | `Float` | `Float/Int` | `Float` 
 <code>+=</code> | addition | `Float/Int` | `Float/Int` | `Float/Int` 
+<code>+=</code> | concatenation | `String` | `Any type` | `String` 
```

Comment:
Please change the `Any type` to "any" (no backticks, see the "Equality operators" table).